src/js/playlist/item.js

```diff
@@ -28,11 +28,6 @@ const Item = function(config) {
         dvrSeekLimit: 25
     }, config);
 
-    if (playlistItem.dvrSeekLimit < 5) {
-        playlistItem.dvrSeekLimit = 5;
-    }
```

Comment:
Don't think this check is needed anymore because it seems to always gets overridden by 'formatItems' in Playlist